src/components/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+interface LinkProps extends React.HTMLProps<HTMLAnchorElement> {
+  children: React.ReactNode;
+  to: string;
+}
+
+const LinkButton: React.FC<LinkProps> = ({ to, children, ...props }) => {
+  return (
+    <>
+      <a
+        href={to}
+        {...props}
+        className="p-2"
+        style={{
+          width: '100px',
+          overflow: 'hidden',
+          whiteSpace: 'nowrap',
+          textOverflow: 'ellipsis',
+          textAlign: 'center',
+        }}
```

Comment:
The same points raised above apply here, please adjust accordingly:

- `HTMLProps` infers too widely, I suggest using `React.ComponentProps<'a'>` as well.
- No need to manually specify `children` and `to` in the type.
- Remove unnecessary fragment.
- `{...props}` should be the last attribute, and no need to specify `href` since it is already part of `props`.
- Use tailwind classes instead of CSS styles. The classes should be the same as for the button since these two components have the same look in the design.

Reply:
Thank You @antosan for the coreections. Making adjustments right away